source/_integrations/home_connect.markdown

```diff
@@ -49,7 +49,7 @@ Note that it depends on the appliance and on API permissions which of the featur
 
 If you don't have [My Home Assistant](/integrations/my) on your installation,
 you can use `<HOME_ASSISTANT_URL>/auth/external/callback` as the redirect URI
-instead.
+instead. But it is important to have the My Home Assistant Url in the redirect Urls list too.
```

Comment:
That statement is incorrect. This particular paragraph is in the section "I have manually disabled My Home Assistant". In those cases, the My Home Assistant URL cannot be used.